sbot/indexfeeds.go

```diff
@@ -0,0 +1,163 @@
+// SPDX-FileCopyrightText: 2021 The Go-SSB Authors
+//
+// SPDX-License-Identifier: MIT
+
+package sbot
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+
+	"go.cryptoscope.co/ssb"
+	refs "go.mindeco.de/ssb-refs"
+)
+
+func newIndexFeedManager(storagePath string) (ssb.IndexFeedManager, error) {
+	m := indexFeedManager{indexes: make(map[string]refs.FeedRef), storagePath: storagePath}
+	// load previous registered indexes (if any)
+	m.load()
+
+	return m, nil
+}
+
+type indexFeedManager struct {
+	// registered indexes
+	indexes     map[string]refs.FeedRef
+	storagePath string
+}
+
+func constructIndexKey(author refs.FeedRef, msgType string) string {
+	return fmt.Sprintf("%s%s", author, msgType)
```

Comment:
```suggestion
	return  author.String() + msgType
```

just a stylisitic thing. also more as an generic advice: fmt format calls are pretty expensive.